src/WebClient.spec.js

```diff
@@ -78,7 +87,7 @@ describe('WebClient', function () {
 
   describe('apiCall()', function () {
     beforeEach(function () {
-      this.client = new WebClient(token, { retryConfig: rapidRetryPolicy });
+      this.client = new WebClient(token, { retryConfig: rapidRetryPolicy, headers: staticHeaders });
```

Comment:
i don't think we should initialize with the static headers for all test cases inside this `describe` block. instead, i think we should add another `describe` block inside this one (as done for other specific features), maybe `describe('when using custom static headers', function () { /* ... */ });` Within that block, we should have at least one test case where a client is constructed with custom static headers, and the request is checked for the existence of those headers (using `nock`). We probably also don't need `staticHeaders` to be declared globally in this file. Let me know if you disagree or if you'd like any further help.